src/Instance.cc

```diff
@@ -95,7 +96,7 @@ GetOtherPid(File &pidFile)
 static bool
 ProcessIsRunning(const pid_t pid)
 {
-    const auto result = kill(pid, 0);
```

Comment:
Please continue to use the POSIX API equivalent call.
```
    const auto result = xkill(pid, 0);
```

Alternatively, if you really want to use the `IsPidValid` name for some reason. Move the custom implementation of that function here as a `static` local function which alternatively implements the Windows logic or calls `xkill` as-needed.